src/libraries/System.Private.CoreLib/src/System/Numerics/Vector_1.cs

```diff
@@ -23,11 +23,8 @@ namespace System.Numerics
     *
     * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * */
 
-    /// <summary>
-    /// A structure that represents a single Vector. The count of this Vector is fixed but CPU register dependent.
-    /// This struct only supports numerical types. This type is intended to be used as a building block for vectorizing
-    /// large algorithms. This type is immutable, individual elements cannot be modified.
-    /// </summary>
+    /// <summary>Represents a single vector of a specified numeric type that is suitable for low-level optimization of parallel algorithms.</summary>
+    /// <typeparam name="T">The vector type. <typeparamref name="T" /> can be any primitive numeric type.</typeparam>
```

Comment:
nit: `T` isn't the vector type, but rather `The type of the elements in the vector`.

I'm not sure `any primitive numeric type` is the right distinguisher either. What people consider "primitive" can vary based on language and familiarity with the underlying runtime.